src/test/java/giis/eshopcontainers/e2e/functional/common/BaseLoggedClass.java

```diff
@@ -173,15 +175,15 @@ protected void logout() throws ElementNotFoundException {
      * Checks if the database migration is complete by attempting to connect and query the number
      * of databases in the master.sys.databases table. Retries the connection and query up to a
      * specified number of times. The number of expected databases are 2 (default) + another 5 databases
-     * created by the different services
+     * created by the different services, so a minimal of 7 tables are expected.
      */
-    protected static void checkDBMigration() throws SQLDataException {
+    protected static void checkDBMigration() {
         // Get properties
         String user = properties.getProperty("SQLDB_USER");
         String password = properties.getProperty("SQLDB_PASSWORD");
         String host = "sqldata_" + tJobName;
-        // Build JDBC URL
-        final int MAX_TABLES = 6;
+        // Minimal amount of tables expected in the database
+        final int MIN_TABLES = 6;
```

Comment:
The comment is not explaining why. Another comment before mentions seven databases, however, here, the variable says six, and it refers to tables, not databases.

Reply:
I've aligned the two comments and the conditional @javiertuya